docs/markdown/bar-series.md

```diff
@@ -117,3 +117,7 @@ The opacity for all elements in the series, this property will be over-ridden by
 #### stroke
 Type: `string|number`
 The outer color for all elements in the series, this property will be over-ridden by color specified in the data attribute.
+
+### style
+Type: `object`
+A list of CSS properties to style the series outside of the explicitly set properties. Note that it will override all other properties (ie fill, stroke, opacity, color).
```

Comment:
I don't agree with the use of the word "CSS" here, "style properties" would be more fitting in my opinion.
Also, we should have one consistent definition and not attempt to change it between components. I'm not talking about possible different examples between the components.

The only exception to this would be the axes one (and other ones below I haven't saw yet that function the same way). One other thing I would say would be that in the case of axe, rename the `style` prop into something else, like `styles`. This way, it will allow users to tell immediately if it's a simple style, or an object containing multiple.